### PR TITLE
docs: Add guidelines and update docs for published library from UI

### PR DESCRIPTION
**Pull request by ad1992**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,23 +12,21 @@
 - Open the library menu (by clicking on the "library" button or typing `9`)
 - The selected object should appear in a square. Click on it (you will see a `+` sign when hovering over it)
 
-4. Still in the library menu, click on "Export". You will get a `.excalidrawlib` file with all your added objects in it.
+4. Still in the library menu, select the libraries you want to publish (Hover over the library items and you should see a checkbox, clicking on which will select the library item).
```
Comment (dwelle) [inline README.md:15]:
```suggestion
4. Still in the library menu, select the libraries you want to publish (hover over a library item and you should see a checkbox, clicking on which will select the library item).
```

Comment (dwelle) [inline README.md:15]:
```suggestion
4. Still in the library menu, select the libraries you want to publish (hover over a library item and you should see a checkbox, clicking on which will select the library item).
```

File: README.md [hunk 2]
```diff
@@ -12,23 +12,21 @@
...
 
-![Gif showing how to add a single object and export the library](public/add_to_library_and_export.gif)
+5. Hit the publish button.
```
Comment (dwelle) [inline README.md:17]:
```suggestion
5. Hit the publish button (in blue), and fill out the details.
```

File: README.md [hunk 3]
```diff
@@ -12,23 +12,21 @@
...
-   2. Upload the screenshot of the preview (as landscape PNG with max width 800px)
-3. Edit the [`libraries.json`](libraries.json) and add your entry to the end
-4. Create a Pull Request to this repo
+-  All library items should be generic and reusable.
+- Don't republish (copy/paste) items from other libraries without making any significant changes.
+- Each item should stand on its own and add value.
+Eg Adding single square or a squiggly line doesn't meet standard.
 
-Once reviewed and approved your library will appear on the [website](https://libraries.excalidraw.com), but keep in mind your libraries have to be for wide usage and not specific to your needs.
+Check out the published [libraries](https://libraries.excalidraw.com/?theme=light&sort=default) to get started.
```
Comment (dwelle) [inline README.md:30]:
```suggestion
Check out our existing [libraries](https://libraries.excalidraw.com/?theme=light&sort=default) to get started.
```

File: README.md [hunk 4]
```diff
@@ -12,23 +12,21 @@
...
-## Add your own library to the list
+## Guidelines
 
-1. Fork this repo
-2. Create a directory with your `name` in the [`libraries`](libraries) (use `kebab-case` for all files and directories)
-   1. Upload your `.excalidrawlib` files
-   2. Upload the screenshot of the preview (as landscape PNG with max width 800px)
-3. Edit the [`libraries.json`](libraries.json) and add your entry to the end
-4. Create a Pull Request to this repo
+-  All library items should be generic and reusable.
```
Comment (dwelle) [inline README.md:25]:
```suggestion
- All library items should be reusable. Not something that only you or your friend will use.
```

Comment (ad1992) [reply README.md:25]: > Not something that only you or your friend will use.
Probably need to reword this.
How about ?
```
All library items should be reusable and not just limited to your usecase
```

File: README.md [hunk 5]
```diff
@@ -12,23 +12,21 @@
...
+- Don't republish (copy/paste) items from other libraries without making any significant changes.
+- Each item should stand on its own and add value.
+Eg Adding single square or a squiggly line doesn't meet standard.
```
Comment (dwelle) [inline README.md:28]:
```suggestion
- In general, each item should be usable and stand on its own.
- Don't publish items which are very easy to create directly within the editor. For example, a single arrow or a square.
```

File: README.md (hunk 4 above)
Comment (dwelle) [reply README.md:25]: Yeah, I updated the change

Comment (ad1992) [reply README.md:25]: ok somehow I am not getting the option to commit it so I will push it

Comment (ad1992) [reply README.md:25]:
```suggestion
- All library items should be reusable and not just limited to your personal use case.

File: README.md [hunk 6]
```diff
@@ -12,23 +12,22 @@
...
-2. Create a directory with your `name` in the [`libraries`](libraries) (use `kebab-case` for all files and directories)
-   1. Upload your `.excalidrawlib` files
-   2. Upload the screenshot of the preview (as landscape PNG with max width 800px)
-3. Edit the [`libraries.json`](libraries.json) and add your entry to the end
-4. Create a Pull Request to this repo
+- All library items should be reusable and not just limited to your personal use case.
+- Don't republish (copy/paste) items from other libraries without making any significant changes.
+- In general, each item should be usable and stand on its own.
+- Don't publish items which are easy to create directly. For example, a single arrow or a square.
+- Don't publish items which are easy to create directly. For example, a single arrow or a square.
```
Comment (ad1992) [inline README.md:29]:
```suggestion
```

**Review by dwelle — APPROVED** [review]

**Comment by sohailamjad256** [comment]
Hi @ad1992 ..i just want to sync excalidraw  add to library api to our excalidraw component..aur any alternatives is there??

**Pull request merged**